### PR TITLE
create Inbox component with list of message and unit-tests

### PR DESCRIPTION
**Pull request by tanyaivattt**

### CODE REVIEW

**Comment by tanyaivattt** [comment]
for testing you should put <InboxComponent /> into mainPage.tsx

File: src/components/inboxComponent/inboxMessageComponent/_test_/inboxMessageComponent.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import '@testing-library/jest-dom';
+//import userEvent from '@testing-library/user-event';
+
+import InboxMessageComponent from '../inboxMessageComponent';
+
+describe('InboxMessageComponent Component', () => {
+  beforeEach(() => {
+    render(
+      <MemoryRouter>
+        <InboxMessageComponent id="" title="" description="" />
```
Comment (Vlad-2109) [inline src/components/inboxComponent/inboxMessageComponent/_test_/inboxMessageComponent.spec.tsx:12]: Do you need these props? Or you can delete them?

File: index.html [hunk 2]
```diff
@@ -2,6 +2,9 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
```
[the remark this answers is not in the document]
Comment (tanyaivattt) [reply index.html:7]: As far as i know, this is the only way to use custom fonts in MUI.
Other ways do not provide such stylization.

File: src/components/inboxComponent/inboxMessageComponent/_test_/inboxMessageComponent.spec.tsx (hunk 1 above)
Comment (tanyaivattt) [reply src/components/inboxComponent/inboxMessageComponent/_test_/inboxMessageComponent.spec.tsx:12]: These props are nessesary according to the syntax 
image

**Review by Vlad-2109 — DISMISSED** [review]

File: src/types/types.ts [hunk 3]
```diff
@@ -45,3 +45,10 @@ export interface ICourseDetailsResponseData{
   completedChapterLength: number,
   chaptersLength: number,
 }
+
+export interface IMessageResponseData{
```
Comment (Virsta) [inline src/types/types.ts:49]: Why do you create another IMessageResponseData if you already have MessageProps in componentType.tsx?

Comment (tanyaivattt) [reply src/types/types.ts]: fixed

**Review by Virsta — APPROVED** [review]

**Review by Vlad-2109 — APPROVED** [review]

**Pull request merged**